app/views/root_taxons/show.html.erb

```diff
@@ -1,12 +1,24 @@
 <%= display_header title: @content_item.title,
-                   breadcrumbs: [:root_taxons, @content_item.title] %>
+                   breadcrumbs: [:root_taxons, @content_item.title] do %>
+
+  <% %w[list bubbles].each do |viz| %>
+    <a href="<%= root_taxon_path(params[:id], viz: viz) %>"
+       class="btn btn-lg <%= viz == params[:viz] ? 'btn-primary' : 'btn-default' %>"
+       role="button">
+      <%= viz.titlecase %>
+    </a>
+  <% end %>
+<% end %>
+
+<%= render params[:viz] if params[:viz].in?(%w[list bubbles])%>
```

Comment:
It's a little bit icky accessing the params hash in the view - could this be defined as an instance variable in the controller instead?

Reply:
I've done some restructuring around the viz parameter in 85e54f3